### DISCUSSION TITLE
[ ] returning around number returned by binding

### DISCUSSION THREAD

**Discussion by timmehrees** [Budibase/budibase#6041] [Help]
Hi All,

I'm using the URL.id binding a fair bit when changing screens, saving data, etc etc. I've noticed that sometimes the URL comes up with /screenname/8 and other times /screenname/[8]

Weirdly, on some screens [8] returns the right record, on others, it doesn't. 

It seems that any of the automatically generated screens work with [8]. 

My current screen I am getting the "id" from has the ID in a form field (returned by an options menu). Is there a way to strip the [ ] from the returned value using handlebars or JS? (I'm not overly familiar with either, and have learnt a lot in the last 2 weeks of working on Budibase).

Thanks.

**Comment by melohagan** [comment]
Hi @timmehrees 

Are the usages of the URL.id consistent across your app? If you remove bindings and re-add them does that persist? 

The values shouldn't have square brackets around them. What version of Budibase is your app running?

Would you be able to send your app export to support@budibase.com ?

**Reply by timmehrees** [reply]
Hi @melohagan 

App export is on its way. Version is 1.0.110, with MySQL (latest docker image....)

The latest binding I added today, and re-added when I thought it wasn't working (until I looked at the URL). In short, it is a number field, that is used by the relationship function, I use a button to open the URL with that field as the id, and it comes up as [8].

The rest of my manually created URL.id references all work normally, but the auto-generated screens, when you first start on the app all seemed to use [8] as the id... 

The only other idea I've had is to do with the relational links?

I've only been working with Budibase for around 2 weeks, and my background is not in programming (although I'm decently handy with MSSql from previous SysAdmin work) - so its very possible i've made a massive boo boo, and totally missed it!

Thanks for your help,
Tim

**Reply by melohagan** [reply]
Thanks @timmehrees 

From what I can see, the link URLs you have look fine. 

I had another issue recently around URL ids, so I think this is a bug of some sorts. It affects postgres as well.
